src/node/routes/index.ts

```diff
@@ -129,6 +129,11 @@ export const register = async (app: App, args: DefaultedArgs): Promise<Disposabl
     express.static(rootPath, {
       cacheControl: commit !== "development",
       fallthrough: false,
+      setHeaders: (res, path, stat) => {
+        if (path.endsWith("/serviceWorker.js")) {
+          res.setHeader("Service-Worker-Allowed", "/")
+        }
+      },
```

Comment:
question: what's this for?

Reply:
The service worker is served from `/_static/out/browser/serviceWorker.js` so by default it is not allowed to register a scope above `/_static/out/browser`.  This lets it register on the root.

Reply:
Will add a comment.